pkg/normalize/validate.go

```diff
@@ -143,6 +144,8 @@ func checkTargets(rec *models.RecordConfig, domain string) (errs []error) {
 		}
 	case "ALIAS":
 		check(checkTarget(target))
+	case "SRV":
+		check(checkTarget(target))
```

Comment:
Is a target of an IP address valid for SRV records? 

If so, I'm not sure I like how this validates. 

`SRV("@", 5, 5, 4242, "2.3.4.5")` is not valid but `SRV("@", 5, 5, 4242, "2.3.4.5.")` would be. But I don't know the rules. 

Should maybe the `checkTarget` function do more to make sure you are *not* pointing CNAME/MX/ALIAS/SRV at raw IP addresses?